prov/gni/src/gnix_ep.c

```diff
@@ -113,21 +113,19 @@ static inline ssize_t __ep_recv(struct fid_ep *ep, void *buf, size_t len,
 
 static inline ssize_t __ep_recvv(struct fid_ep *ep, const struct iovec *iov,
 				 void **desc, size_t count, fi_addr_t src_addr,
-				 void *context, uint64_t flags, uint64_t tag,
-				 uint64_t ignore)
+				 void *context, uint64_t flags, uint64_t tag)
 {
 	struct gnix_fid_ep *ep_priv;
 
-	if (!ep || !iov || count != 1) {
+	if (!ep || !iov || !count) {
```

Comment:
Maybe the following would be appropriate in the event that someone does something silly? 

```C
if (count == 1)
    return __gnix_recv(...)
```